src/Windowing/iOS/Silk.NET.Windowing.Sdl.iOS/Class1.cs

```diff
@@ -46,7 +45,7 @@ public static unsafe void RunApp(int numArgs, byte** args, MainFunction callback
             EndRun();
         }
 
-        public static unsafe void RunApp(int numArgs, byte** args, Action<string[]> callback)
+        public static unsafe void RunApp(int numArgs, byte** args, System.Action<string[]> callback)
```

Comment:
Why?
Also, the file is called `Class1`

Reply:
Because on iOS `System` has a type called `nint` so we couldn't use `nint` while also importing `System`.

And yeah I know about the file, will need to fix that at some point.